### ISSUE TITLE
SAR accuracy results is misunderstanding

### ISSUE THREAD

**Issue by andyjpaddle** [open-mmlab/mmocr#424]
I have used the pre-trained model released by this repo to eval the public OCR dataset. The eval log gives some results, including: word_acc, word_acc_ignore_case, word_acc_ignore_case_symbol, etc. Which result is the reported result? It seems like the "word_acc_ignore_case_symbol " is the final results, can you tell me the reason? Besides, the "word_acc " is so low, however, the word_acc  is always used to be the final results.

**Comment by cuhk-hbsun** [comment]
Thanks for your feedback. 
1. For academic evaluation, the word accuracy is reported under the condition: case insensitive and ignoring tokens that are not a number or English characters. You can check this in the papers.
2. In our metric, it is `word_acc_ignore_case_symbol`. 
3. Different methods support differnet number of classes(tokens). For example, `SAR` is `DICT90` while `CRNN` is `DICT36`. But for evaluation metric, they are the same.

**Comment by andyjpaddle** [comment]
OK，thanks for that. You mean the `word_acc_ignore_case_symbol` is the results that removing some tokens that are not a number or English characters.

**Comment by cuhk-hbsun** [comment]
> OK，thanks for that. You mean the `word_acc_ignore_case_symbol` is the results that removing some tokens that are not a number or English characters.

Yeah

**Comment by andyjpaddle** [comment]
> > OK，thanks for that. You mean the `word_acc_ignore_case_symbol` is the results that removing some tokens that are not a number or English characters.
> 
> Yeah

ok , thank you very much!

**Comment by binzh93** [comment]
> Thanks for your feedback.
> 
> 1. For academic evaluation, the word accuracy is reported under the condition: case insensitive and ignoring tokens that are not a number or English characters. You can check this in the papers.
> 2. In our metric, it is `word_acc_ignore_case_symbol`.
> 3. Different methods support differnet number of classes(tokens). For example, `SAR` is `DICT90` while `CRNN` is `DICT36`. But for evaluation metric, they are the same.

Different alg training on `DICT90`  or `DICT36`,  evaluation all on `DICT36`?

**Comment by Mountchicken** [comment]
> > Thanks for your feedback.
> > 
> > 1. For academic evaluation, the word accuracy is reported under the condition: case insensitive and ignoring tokens that are not a number or English characters. You can check this in the papers.
> > 2. In our metric, it is `word_acc_ignore_case_symbol`.
> > 3. Different methods support differnet number of classes(tokens). For example, `SAR` is `DICT90` while `CRNN` is `DICT36`. But for evaluation metric, they are the same.
> 
> Different alg training on `DICT90` or `DICT36`, evaluation all on `DICT36`?

Evaluating on `DICT36` is common sense in the academic community because it can get a higher accuracy.
